matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/crypto/RoomCryptoService.kt

```diff
@@ -29,7 +29,7 @@ interface RoomCryptoService {
     /**
      * Enable encryption of the room
      */
-    suspend fun enableEncryption(algorithm: String = MXCRYPTO_ALGORITHM_MEGOLM)
+    suspend fun enableEncryption(algorithm: String = MXCRYPTO_ALGORITHM_MEGOLM, force: Boolean = false)
```

Comment:
Maybe add comment to tell what will be the behavior of `force`?

Reply:
added